docs/index.html

```diff
@@ -63,8 +63,7 @@
 <div class="doc-link">
   <a href="./model.html">Model Schema</a><br>
   <a href="./simulation-parameter.html">Energy Simulation Parameter Schema</a><br>
-  <a href="./sensor-grid.html">Radiance Sensor Grid Schema</a><br>
-  <a href="./view.html">Radiance View Schema</a><br>
+  <a href="./radiance.html">Radiance Simulation Objects Schema</a><br>
```

Comment:
Again, I think "radiance-asset.html" and "Radiance Simulation Asset Schema" is clearer.